readme.md

```diff
@@ -167,6 +167,24 @@ For fields with _type_: "select", another field _foreign_ is mandatory. This fie
 ]
 ```
 
+###MultiSelect Field
```

Comment:
Please also add these field type in the "fields" section of readme as well

Reply:
Added